src/utils/applescript.js

```
@@ -0,0 +1,361 @@
+          try {
+            // 制御文字を含むJSONを修正
+            const cleanedOutput = output.replace(/[\u0000-\u001F\u007F-\u009F]/g, '');
+            const result = JSON.parse(cleanedOutput);
+            resolve(result);
+          } catch (innerError) {
+            console.error(`Error parsing JSON with control characters:`, innerError);
+
+            // 制御文字を含むJSONを別の方法で修正
+            const sanitizedOutput = output
+              .replace(/[\u0000-\u0008\u000B\u000C\u000E-\u001F\u007F-\u009F]/g, '')
+              .replace(/\r\n|\n|\r/g, '\\n')
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)